### ISSUE TITLE
AVT 1 - Default Button has DAP violations when disabled knob is used

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4524]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React Component
Run DAP and the following violations are present (see screenshot):
<img width="947" alt="Screen Shot 2019-11-01 at 11 23 59 AM" src="https://user-images.githubusercontent.com/21676914/68040445-54d73b00-fc9c-11e9-8d62-1971fff2eefb.png">

Note: The violations are only present in the default button when the disable knob is selected using any button kind on knob (i.e. Primary, Secondary, Danger, Ghost)

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
